lib/providers/livesync-provider.ts

```diff
@@ -45,10 +45,10 @@ export class LiveSyncProvider implements ILiveSyncProvider {
 		await this.$platformService.buildPlatform(device.deviceInfo.platform, buildConfig, projectData);
 		let platformData = this.$platformsData.getPlatformData(device.deviceInfo.platform, projectData);
 		if (device.isEmulator) {
-			return this.$platformService.getLatestApplicationPackageForEmulator(platformData).packageName;
+			return this.$platformService.getLatestApplicationPackageForEmulator(platformData, { isReleaseBuild: buildConfig.release }).packageName;
 		}
 
-		return this.$platformService.getLatestApplicationPackageForDevice(platformData).packageName;
+		return this.$platformService.getLatestApplicationPackageForDevice(platformData, { isReleaseBuild: buildConfig.release }).packageName;
```

Comment:
Do you think we can directly pass on the `buildConfig` property, so that we have it in handy in case we need other property in the future?